cmd/nitro/init.go

```diff
@@ -937,6 +945,22 @@ func openInitializeExecutionDB(ctx context.Context, stack *node.Node, config *No
 	return rebuildLocalWasm(ctx, &config.Execution, l2BlockChain, executionDB, wasmDB, config.Init.RebuildLocalWasm)
 }
 
+func resolveInitialL1BaseFee(genesisArbOSInit *params.ArbOSInit, l2Config *conf.L2Config) (*big.Int, error) {
+	parsedInitialL1BaseFee, err := l2Config.InitialL1BaseFeeParsed()
+	if err != nil {
+		return nil, err
+	}
+
+	if genesisArbOSInit == nil {
+		return parsedInitialL1BaseFee, nil
+	}
+	genesisFee := genesisArbOSInit.GetInitialL1BaseFee()
+	if l2Config.InitialL1BaseFee != "" && parsedInitialL1BaseFee.Cmp(genesisFee) != 0 {
+		return nil, fmt.Errorf("initial l1 base fee configuration mismatch: `genesis-json-file` sets the value to %s, while `initial-l1base-fee` was set to %s", genesisFee.String(), l2Config.InitialL1BaseFee)
```

Comment:
nitpick:
```suggestion
		return nil, fmt.Errorf("initial l1 base fee configuration mismatch: `genesis-json-file` sets the value to %s, while `initial-l1base-fee` flag was set to %s", genesisFee.String(), l2Config.InitialL1BaseFee)
```

Reply:
done in [41de374](https://github.com/OffchainLabs/nitro/pull/4171/commits/41de374d40d284e4c82bdb98d9753e4198146573)